### DISCUSSION TITLE
New Hackaday article about  USB-C  implementation done in Pinecil V2

### DISCUSSION THREAD

**Discussion by River-Mochi** [Ralim/IronOS#1551] [Ideas]
New article about USB-C and how it is implemented in Pinecil V2. they are using  photos of the older  V1 model.
https://hackaday.com/2023/01/31/all-about-usb-c-pinecil-soldering-iron

**Comment by Ralim** [comment]
The photos are valid for their talking points either way; and there are more V1 images licensed out there I'm guessing.
